pkgs/tools/misc/toybox/default.nix

```diff
@@ -18,7 +18,8 @@ stdenv.mkDerivation rec {
   };
 
   depsBuildBuild = [ buildPackages.stdenv.cc ]; # needed for cross
-  buildInputs = lib.optionals enableStatic [ stdenv.cc.libc stdenv.cc.libc.static ];
+  buildInputs = lib.optionals (enableStatic && stdenv.cc.libc ? static)
+    [ stdenv.cc.libc stdenv.cc.libc.static ];
```

Comment:
```suggestion
```
I think we can just remove this entirely? Seems weird to specify libc at all, it should just be part of cc afaik.

Reply:
Nope, removing either element breaks static glibc builds with `toybox.override { enableStatic = true; }`.

Reply:
Ah I see, thank you